similarity/config/datastore.properties

```diff
@@ -0,0 +1,3 @@
+ENDPOINT	http://smaug.cs.illinois.edu:8080
```

Comment:
This happens because bucket's policy is not set. I suspect you created the bucket manually. I  had to fixed it. This works for me now: 
```
Datastore ds = new Datastore("http://smaug.cs.illinois.edu:8080")
File f = ds.getFile("org.cogcomp.wordembedding", "phrase2vec.txt", 1.5);
```